### ISSUE TITLE
the  pretrain_models of CrowdPose data

### ISSUE THREAD

**Issue by suijua** [buptxyb666/AdaptivePose#14]
hi, can you provide the  pretrain_models of CrowdPose data, I train the model by pose_hrnet_w32_384x288.pth and the outcome is not great ,thank you!

**Comment by buptxyb666** [comment]
OK, due to the change of variable name, the previous pretrain models of CrowdPose is unavailable. I will provide the new model in the last few days.

**Comment by buptxyb666** [comment]
I found some issues in CrowdPose training code in current version:
1. Please make sure that the train and val set are both used for training in /opt/tiger/adaptivepose/src/lib/datasets/dataset/crowdpose.py. 
2. Please make sure that the correct augmentation for the samples in train and val set.
/opt/tiger/adaptivepose/src/lib/datasets/sample/multi_pose_crowdpose.py

I will update the related code for crowdpose training.

**Comment by suijua** [comment]
> CrowdPose training code in current version:

ok, thanks!

**Comment by buptxyb666** [comment]
The results reported in paper are trained by hrnetv2_w32_imagenet_pretrained.pth. I think using pose_hrnet_w32_384x288.pth will achieve better performance. We also try to use OKS loss in crowdpose. Working in progress.

**Comment by buptxyb666** [comment]
Your code for CrowdPose maybe has some problems. I get 62.0 AP on CrowdPose test set at 53 epoch now(single-scale test without flip, --keep-res should be added).  bs=64 for 8-v100, lr=2.5e-4.

**Comment by buptxyb666** [comment]
The related code for crowdpose training and weights are updated. Sorry for the delay.

**Comment by suijua** [comment]
> The related code for crowdpose training and weights are updated. Sorry for the delay.
Ok，thank you for your work!
